src/Unsafe/Unique/Tag.hs

```diff
@@ -11,6 +11,7 @@ import Unsafe.Unique.Prim
 import Unsafe.Coerce
 import Control.Monad.Primitive
 import Control.Monad
+import Data.Typeable ((:~:)(..))
```

Comment:
This is slightly inconsistent: `(:~:)(..)` is imported from `Data.Type.Equality` in `Data.Unique.Tag`.